python/tests/test_config.py

```diff
@@ -151,6 +151,33 @@ def test_biophysical_properties_raises(self):
             }
         self.assertRaises(SonataError, CircuitConfig, json.dumps(contents), PATH)
 
+    def test_vasculature_properties(self):
+        contents = {
+            "manifest": {
+                "$BASE_DIR": "./",
+                },
+            "components": {
+                "vasculature_mesh": "$BASE_DIR/vasculature_mesh.h5",
+            },
+            "networks": {
+                "nodes": [
+                    {
+                        "nodes_file": "$BASE_DIR/nodes1.h5",
+                        "populations": {
+                            "nodes-A": {
+                                "type": "vasculature",
+                                "vasculature_file": "$BASE_DIR/vasculature_file.h5",
```

Comment:
The `vasculature_mesh` is below `vasculature_file` in ngv configs. The components are not usually used.

Reply:
yeah, I'm checking the resolution works, if it is in the config.